<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MooovR</class>
 <widget class="QMainWindow" name="MooovR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MooovR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Paste destination paths</string>
    </property>
    <widget class="QTextEdit" name="dstF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Source files/folders</string>
    </property>
    <widget class="QTextEdit" name="srcF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_moving">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="mooovr_progress">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
